pkg/proxy/iptables/proxier_test.go

```diff
@@ -2209,7 +2209,96 @@ func TestClusterIPEndpointsJump(t *testing.T) {
 	})
 }
 
-func TestOpenShiftDNSHack(t *testing.T) {
+func TestOpenShiftDNSHackTCP(t *testing.T) {
```

Comment:
this shouldn't be needed... `assertIPTablesRulesEqual` is supposed to deal with rule sorting and stuff